packages/graphback-datasync/src/providers/DatasyncMongoDBDataProvider.ts

```diff
@@ -76,7 +76,7 @@ export class DataSyncMongoDBDataProvider<Type = any> extends MongoDBDataProvider
     const projection = this.buildProjectionOption(context);
     const query = this.db.collection(this.collectionName).findOne({
       ...filter,
-      _deleted: false
+      _deleted: { $ne: true}
```

Comment:
I guess we want to cather situation when `_deleted` doesn't exist?

Reply:
Yes :+1: